website/docs/language/modules/sources.mdx

```diff
@@ -315,39 +315,39 @@ module "vpc" {
 
 ## HTTP URLs
 
-When you use an HTTP or HTTPS URL, OpenTF will make a `GET` request to
+When you use an HTTP or HTTPS URL, OpenTofu will make a `GET` request to
 the given URL, which can return _another_ source address. This indirection
 allows using HTTP URLs as a sort of "vanity redirect" over a more complicated
 module source address.
 
-OpenTF will append an additional query string argument `opentf-get=1` to
+OpenTofu will append an additional query string argument `tofu-get=1` to
 the given URL before sending the `GET` request, allowing the server to
-optionally return a different result when OpenTF is requesting it.
+optionally return a different result when OpenTofu is requesting it.
 
-If the response is successful (`200`-range status code), OpenTF looks in
+If the response is successful (`200`-range status code), OpenTofu looks in
 the following locations in order for the next address to access:
 
-- The value of a response header field named `X-OpenTF-Get`.
+- The value of a response header field named `X-OpenTofu-Get`.
```

Comment:
Same

Reply:
This is still X-Terraform-Get in the code right now.